scripts/build/termux_step_get_dependencies.sh

```diff
@@ -120,7 +93,7 @@ termux_force_check_package_dependency() {
 
 termux_run_build-package() {
```

Comment:
There's some mixed indentation in this function on the last two lines of the function body.

Reply:
Fixed.